packages/mocks/src/getMockData.ts

```diff
@@ -1,13 +1,9 @@
-import { PathParams } from 'msw';
-
-import * as providers from './data/provider.json';
```

Comment:
if you are not using the json file somewhere else you can delete it, I only added it as an example hoe to quickly record data from the browser and use it as mock data